index.d.ts

```diff
@@ -29,7 +29,7 @@ export type ValidatorCompiler = (
 
 export declare function ValidatorSelector(): ValidatorCompiler;
 
-export type { Options } from "ajv";
+export type { Options, ErrorObject } from "ajv";
```

Comment:
Could you add a test to this file https://github.com/fastify/ajv-compiler/blob/main/test/types/index.test.ts?

Reply:
I can't figure out how to make a type test that is exported from the same module

Reply:
It should be doable by importing the same from both the modules and `expectType<ErrorFromThisModule>(ErrorImportedFromAjv)`

By doing this we assure that we don't remove the Error exportorted types

Reply:
```typescript
expectType<unknown>(({} as ErrorObject).data)
expectType<string>(({} as ErrorObject).instancePath)
expectType<string>(({} as ErrorObject).keyword)
expectType<string | undefined>(({} as ErrorObject).message)
expectType<Record<string, any>>(({} as ErrorObject).params)
expectType<AnySchemaObject | undefined>(({} as ErrorObject).parentSchema)
expectType<string | undefined>(({} as ErrorObject).propertyName)
expectType<unknown>(({} as ErrorObject).schema)
expectType<string>(({} as ErrorObject).schemaPath)
```

Reply:
but this module must have import the type from `ajv` or construct in this module (which is not possible) and compare the type from 'ajv' and exported

Sorry, I don't understand how to write it...

Reply:
> ```ts
> expectType<unknown>(({} as ErrorObject).data)
> expectType<string>(({} as ErrorObject).instancePath)
> expectType<string>(({} as ErrorObject).keyword)
> expectType<string | undefined>(({} as ErrorObject).message)
> expectType<Record<string, any>>(({} as ErrorObject).params)
> expectType<AnySchemaObject | undefined>(({} as ErrorObject).parentSchema)
> expectType<string | undefined>(({} as ErrorObject).propertyName)
> expectType<unknown>(({} as ErrorObject).schema)
> expectType<string>(({} as ErrorObject).schemaPath)
> ```

can I take this code?

Reply:
Sure.